Common/Product/TestAdapter/VisualStudioApp.cs

```diff
@@ -119,7 +118,7 @@ private static DTE GetDTE(int processId)
                         // Do nothing, there is something in the ROT that we do not have access to.
                     }
 
-                    if (!string.IsNullOrEmpty(name) && string.Equals(name, progId, StringComparison.Ordinal))
+                    if (!string.IsNullOrEmpty(name) && !StringComparer.Ordinal.Equals(name, progId))
```

Comment:
I think this is still off, no?  It shouldn't have the ! for StringComparer.Ordinal.Equals(name, progId))